osf/models/registrations.py

```diff
@@ -407,7 +422,7 @@ def terminate_embargo(self, forced=False):
         if not self.is_embargoed:
             raise NodeStateError('This node is not under active embargo')
 
-        action = NodeLog.EMBARGO_TERMINATED if forced else NodeLog.EMBARGO_COMPLETED
+        action = NodeLog.EMBARGO_COMPLETED if not forced else NodeLog.EMBARGO_TERMINATED
```

Comment:
This should be changed back

```suggestion
        action = NodeLog.EMBARGO_TERMINATED if forced else NodeLog.EMBARGO_COMPLETED
```